src/util/commandline/command.ts

```diff
@@ -194,4 +195,17 @@ export class Command {
 
     return input;
   }
+
+  private checkForUpdates(): void {
```

Comment:
Good code, but it doesn't belong here. This check should be performed in bin/appcenter.js instead. That's the main entry point.

It should also listen for the `--quiet` switch and not print anything if the "quiet" switch is turned on. Otherwise, automation scripts will randomly start failing due to extra unexpected output.

Reply:
Changed. Please have a look at new iteration.